src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -404,7 +418,23 @@ class QueryAppsDataValidator {
     return true;
   }
 
-  const smart_objects::SmartObject& data_;
+  bool FindMandatoryParameters(
```

Comment:
@Kozoriz Probably function's logic does not correspond its name. Suggest to rename it to something like: `CheckMandatoryParametersPresent` or `MandatoryParametersPresent` ...